<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Pipe_Segement_Details</class>
 <widget class="QWidget" name="Pipe_Segement_Details">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>228</width>
    <height>181</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>221</width>
     <height>148</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Internal Diameter</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="D_le"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Roughness</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Length</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="e_le"/>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="L_le"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Total Value of k</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="k_le"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pipe_sgmnt_pb">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>150</y>
     <width>90</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>D_le</tabstop>
  <tabstop>e_le</tabstop>
  <tabstop>L_le</tabstop>
  <tabstop>k_le</tabstop>
  <tabstop>pipe_sgmnt_pb</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
